packages/block-library/src/cover/save.js

```diff
@@ -91,6 +94,10 @@ export default function save( { attributes } ) {
 
 	const gradientValue = gradient || customGradient;
 
+	if ( useFeaturedImage ) {
```

Comment:
We should include a comment here to explain what is going on.

Reply:
This is no longer implemented.